### DISCUSSION TITLE
Using lazygit in lazyvim is affected by keymaps.

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4066] [Q&A]
When I open lazygit in lazyvim and move commit using `<C-j>` `<C-k>`, the cursor will move away from lazygit. 

I suspect that the keymaps in lazyvim affect lazygit, is there any way to block this effect?

**Comment by dpetka2001** [comment]
Change keymaps in LazyVim or change keymaps in lazygit or just unmap the mappings mentioned [here](https://github.com/LazyVim/LazyVim/blob/a1335e59e15ef55ee43dd30120da948bc0f293c3/lua/lazyvim/config/keymaps.lua#L169-L172) from terminal mode only (that also means that in terminal mode you would have to use `<C-w>j` instead). I fail to think of another solution.

**Reply by iamxiaojianzheng** [reply]
Is it possible to disable the original mapping after opening terminal?

**Reply by dpetka2001** [reply]
You could try with an autocmd (probably on `TermOpen` event), but never tried that, so can't say for 100%.

**Comment by folke** [comment]
LazyVim disables already disables those keys when using lazgit.

Opening lazygit sets this option: https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/util/terminal.lua#L69

**Reply by iamxiaojianzheng** [reply]
So why does the cursor still exit lazygit when I use `<C-j>` ? 😢

**Reply by folke** [reply]
because you open a temrinal and start lazygit there?
If you use the lazygit keymap, it shouldn't move

**Reply by iamxiaojianzheng** [reply]
get it.
